uploadComplete.php

```diff
@@ -406,35 +429,51 @@
 		// Query to see if the Title already exists, if so, get the ID
 		#################################################################
 		//first, remove the '-' from the ISSNs
-		$pISSN = strtoupper(trim(str_replace ('-','',$pISSN)));
-		//remove blank
-		$pISSN = strtoupper(trim(str_replace (' ','',$pISSN)));
-		if (strpos(strtoupper($pISSN),'N/A') !== false) $pISSN = '';
-		if ($pISSN == '00000000') $pISSN = '';
-		if (strtoupper($pISSN) == 'XXXXXXXX') $pISSN = '';
-		if (strtoupper($pISSN) == '.') $pISSN = '';
-
-		$eISSN = strtoupper(trim(str_replace ('-','',$eISSN)));
-		//remove blank
-		$eISSN = strtoupper(trim(str_replace (' ','',$eISSN)));
-		if (strpos(strtoupper($eISSN),'N/A') !== false) $eISSN = '';
-		if ($eISSN == '00000000') $eISSN = '';
-		if (strtoupper($eISSN) == 'XXXXXXXX') $eISSN = '';
-		if (strtoupper($eISSN) == '.') $eISSN = '';
-
-		$pISBN = strtoupper(trim(str_replace ('-','',$pISBN)));
-		//remove blank
-		$pISBN = strtoupper(trim(str_replace (' ','',$pISBN)));
-		if (strpos(strtoupper($pISBN),'N/A') !== false) $pISBN = '';
-		if ($pISBN == '00000000') $pISBN = '';
-		if (strtoupper($pISBN) == 'XXXXXXXX') $pISBN = '';
-
-		$eISBN = strtoupper(trim(str_replace ('-','',$eISBN)));
-		//remove blank
-		$eISBN = strtoupper(trim(str_replace (' ','',$eISBN)));
-		if (strpos(strtoupper($eISBN),'N/A') !== false) $eISBN = '';
-		if ($eISBN == '00000000') $eISBN = '';
-		if (strtoupper($eISBN) == 'XXXXXXXX') $eISBN = '';
+		if (isset($pISSN)) {

```

Comment:
This is probably not necessary since you made sure that these variables were set to at least null starting at line 197. However, I don't think there's any harm in checking again.